controls/repository.sp

```diff
@@ -51,10 +52,8 @@ benchmark "repository_checks" {
 benchmark "repository_mod_checks" {
   title = "GitHub Mod Repository Checks"
   children = [
-    # TODO: Re-add control once v3 hydrate data works
-    #control.repository_mod_has_mandatory_topics,
-    # TODO: Re-add control after testing API limits
-    #control.repository_mod_uses_monotonic_versioning,
+    control.repository_mod_has_mandatory_topics,
+    control.repository_mod_uses_monotonic_versioning,
```

Comment:
Can you please leave this control commented out for now? I think we need a better approach than scanning all releases, maybe we should take the last 5 releases, but can make that change later